PC/pyconfig.h

```diff
@@ -72,8 +72,19 @@ WIN32 is still required for the locale module.
 #define USE_SOCKET
 #endif
 
-#if defined(WINAPI_FAMILY) && (WINAPI_FAMILY != WINAPI_FAMILY_DESKTOP_APP)
-#define MS_WINDOWS_NON_DESKTOP
+#include <winapifamily.h>
```

Comment:
what is the policy in regards to included in `pyconfig.h`?

Reply:
I think we should only define the new values if `Py_BUILD_CORE` is defined, which means we only include `winapifamily.h` in that case.

Extension modules/host applications will need to do their own version of checking partitions. Hopefully we aren't changing our public ABI in any way that depends on these variables (changing available Python functions is fine).